ERCS/erc-chain-identifiers.md

```diff
@@ -0,0 +1,106 @@
+---
+eip: 7785
+title: On-chain registration of chain identifiers
+description: Derive chain identifiers by hashing their chain name and use ENS to map chain names to these identifiers
+author: Marco Stronati (@paracetamolo), Jeff Lau <jeff-lau@live.com>
+discussions-to: https://ethereum-magicians.org/t/on-chain-registration-of-chain-identifiers/21299
```

Comment:
@SamWilsn created a discussions topic for the EIP.  You can either get him to point to this ERC or create a new one.

Reply:
I will ask him to update the discussion to the new PR. Thank you.